src/primitive-field/primitive-field.component.html

```diff
@@ -2,7 +2,7 @@
   <div class="editable-field-container" [ngClass]="errorNgClass" [tooltipHtml]="errors | errorsToMessagesHtml" tooltipTrigger="mouseenter"
     [tooltipEnable]="isErrorTooltipEnabled">
     <div *ngSwitchCase="'string'">
-      <textarea rows="1" textareaAutofit [(ngModel)]="value" (blur)="commitValueChange()" (keypress)="onKeypress($event)" placeholder="{{schema.title}}"></textarea>
+      <textarea rows="1" [tabindex]="tabIndex" textareaAutofit [(ngModel)]="value" (blur)="commitValueChange()" (keypress)="onKeypress($event)" placeholder="{{schema.title}}"></textarea>
```

Comment:
I think, all input field here should have `tabIndex`, at least the `searchable-dropdown` and `autocomplete-input.

@jmartinm